test/e2e/specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -278,31 +278,27 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 				);
 			} );
 
-			// Temporarily disabled
-			//
-			// See #40818
-			//
-			// step( 'Can see the Earn blocks', async function() {
-			// 	const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
-			// 	await gEditorComponent.openBlockInserterAndSearch( 'earn' );
-			// 	assert.strictEqual(
-			// 		await gEditorComponent.isBlockCategoryPresent( 'Jetpack' ),
-			// 		true,
-			// 		'Earn (Jetpack) blocks are not present'
-			// 	);
-			// 	await gEditorComponent.closeBlockInserter();
-			// } );
-
-			// step( 'Can see the Grow blocks', async function() {
-			// 	const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
-			// 	await gEditorComponent.openBlockInserterAndSearch( 'grow' );
-			// 	assert.strictEqual(
-			// 		await gEditorComponent.isBlockCategoryPresent( 'Grow' ),
-			// 		true,
-			// 		'Grow (Jetpack) blocks are not present'
-			// 	);
-			// 	await gEditorComponent.closeBlockInserter();
-			// } );
+			step( 'Can see the Earn blocks', async function() {
+				const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
+				await gEditorComponent.openBlockInserterAndSearch( 'earn' );
+				assert.strictEqual(
+					await gEditorComponent.isBlockCategoryPresent( 'Earn' ),
```

Comment:
This is the real change (`Jetpack` changed to `Earn`)

Reply:
Ah, good grab.

Reply:
🤦‍♂ _That's_ what I missed in https://github.com/Automattic/wp-calypso/pull/40818. That's what I get for rushing 🤷‍♂ 

Thanks @jeherve 👍